Sources/XMTP/Messages/Topic.swift

```diff
@@ -39,4 +39,8 @@ public enum Topic {
 	private func wrap(_ value: String) -> String {
 		"/xmtp/0/\(value)/proto"
 	}
+    
+    static func isValidTopic(topic: String) -> Bool {
+        return topic.allSatisfy(\.isASCII)
```

Comment:
also [commented](https://github.com/xmtp/xmtp-android/pull/141#discussion_r1408655561) on the Android PR on this. do we want to allow characters 0-32 in topics?